src/panels/menu/help/HelpMain.tsx

```diff
@@ -1,16 +1,10 @@
 import React from 'react';
-import {ProjectSettings} from "../../../config/Variables";
 import HelpCheatSheet from "./HelpCheatSheet";
-import {helpModal} from "../../../locale/help";
 
 export default class HelpMain extends React.Component {
 
 	render() {
 		return (<div>
-			<p>{helpModal[ProjectSettings.viewLanguage].intro.t2}</p>
-			<p><a href="https://github.com/opendata-mvcr/ontoGrapher/issues/new"
-				  rel="noopener noreferrer"
-				  target="_blank">https://github.com/opendata-mvcr/ontoGrapher/issues/new</a></p>
```

Comment:
TBH it would be best if you kept the link to issues as an `a` element in the new solution, because of two reasons:
1) Users see the `href` in the status bar in the browser
2) Browsers very often block window.open() because it's used by ads etc.

But it's not a big deal. I guess it was impossible to implement with the split button requirement.

Reply:
I managed to refactor the SplitButton component such that it is implemented as a <a> with href (a feature of React-Bootstrap).